.eslintrc.js

```diff
@@ -1,43 +1,34 @@
 module.exports = {
-	'env': {
-		'browser': true,
-		'commonjs': true,
-		'es2021': true,
-		'jest': true
-	},
-	'extends': 'eslint:recommended',
-	'overrides': [
-		{
-			'env': {
-				'node': true
-			},
-			'files': [
-				'.eslintrc.{js,cjs}'
-			],
-			'parserOptions': {
-				'sourceType': 'script'
-			}
-		}
-	],
-	'parserOptions': {
-		'ecmaVersion': 'latest'
-	},
-	'rules': {
-		'indent': [
-			'error',
-			'tab',
-		],
-		'linebreak-style': [
```

Comment:
I think we could just keep this - unless you have a good reason for removing it

Reply:
Yes I have a good reason. I changed it to match the one we have on educado-mobile. The one we used on the backend did not work, while the one used on mobile worked. Therefore, we keep the one that does work :))

Reply:
I don't know if it's important, but unless you have found out that this is what is wrong with a file that does not work, then this is hardly a good reason to delete it...